setup.py

```diff
@@ -38,14 +43,10 @@
               'linter'],
     classifiers=[
         'Development Status :: 4 - Beta',
-##      'Development Status :: 5 - Production/Stable', eventually...
+        # 'Development Status :: 5 - Production/Stable', eventually...
         'Environment :: Console',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: GNU General Public License (GPL)'
-            if licence.startswith('GPL') else
-        'License :: OSI Approved :: MIT License'
-            if licence.startswith('MIT') else
-        'License :: uhh, dunno',  # fail PyPI upload intentionally until fixed
+        'License :: OSI Approved :: MIT License',
```

Comment:
Actually I changed my mind; the dynamic computation is not gaining me anything, and I already do not have a single source of truth given that I'd have to update the entire `LICENSE.txt` in addition to `__licence__` in check_manifest.py.

So this is fine.